systemjs/react-hello-world/index.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SystemJS AMD Modules from CDN Example</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="systemjs-importmap">
+    {
+      "imports": {
+        "react": "https://cdn.jsdelivr.net/npm/react/umd/react.production.min.js",
+        "react-dom": "https://cdn.jsdelivr.net/npm/react-dom/umd/react-dom.production.min.js"
+      }
+    }
+  </script>
+  <script src="https://cdn.jsdelivr.net/npm/systemjs/dist/system.min.js"></script>
+</head>
+<body>
+  <div id="react-root"></div>
+  <script type="systemjs-module" src="./dist/react-hello-world.js"></script>
```

Comment:
It might be better to put this at the top of the page to encourage faster initial loading (right before SystemJS actually).

Reply:
That would work for SystemJS, but not for native modules. Right?

The react-hello-world module needs the `react-root` div to exist when it synchronously executes. Native browser modules would synchronously execute before the rest of the document is processed and dom nodes below the `<script>` would not be created.

My thought was to encourage a "would work with native modules" approach.

Reply:
Native modules are "defer by default". The advised workflow would always be to have a custom domready in your own code though in case scripts are cached and execute before the DOM is loaded. But having scripts load early should be the priority for perf.